packages/api/emails/decline/html.pug

```diff
@@ -1,3 +1,9 @@
+
+
+if(appUrl === 'http://localhost:3000')
+    img(src= "https://ksf-stg.herokuapp.com/email/header.jpg" alt='header' style='width:100%; height:auto;')
+else:
```

Comment:
Can probably undo this. We're handling this in a separate PR.

Reply:
Extra code has been remove from pug file.